realm/realm-library/src/androidTest/java/io/realm/NotificationsTest.java

```diff
@@ -946,20 +946,20 @@ public void run() {
     public void asyncRealmObjectShouldNotBlockBackgroundCommitNotification() {
         final AtomicInteger numberOfRealmCallbackInvocation = new AtomicInteger(0);
         final CountDownLatch signalClosedRealm = new CountDownLatch(1);
-        looperThread.realm.addChangeListener(new RealmChangeListener<Realm>() {
+        final Realm realm = looperThread.realm;
+        realm.addChangeListener(new RealmChangeListener<Realm>() {
             @Override
             public void onChange(Realm object) {
```

Comment:
Can we find a better name? I mean, when you read quickly, you can easily read it as a `RealmObject`.